src/test/java/com/artipie/api/RepositoryVerticleTest.java

```diff
@@ -52,6 +53,7 @@ static void prepare(final Vertx vertx, final VertxTestContext ctx) throws IOExce
             ),
```

Comment:
Test works.
I would add ping of port during some time period, to avoid time consumption of test, because time period may be more or less than 3 seconds, it depends on several conditions. If test ping every half of second and period is shorter than 3 second then test will start quicker.